lib/server.js

```diff
@@ -4,7 +4,7 @@ var Hapi           = require("hapi");
 var Bluebird       = require("bluebird");
 var MongoMapper    = require("genesis").MongoMapper;
 
-var mapper = new MongoMapper(process.env.MONGODB_URL);
+var mapper = new MongoMapper(process.env.MONGODB_URI);
```

Comment:
This seems unrelated to this PR. Is there a reason that this is changing in the PR?